src/platform/plugins/shared/embeddable/public/state_transfer/embeddable_state_transfer.ts

```diff
@@ -146,6 +146,39 @@ export class EmbeddableStateTransfer {
     );
   }
 
+  /**
```

Comment:
Keeping both `getIncomingEmbeddablePackage` and `navigateToWithMultipleEmbeddablePackage` exposes tech debt and puts historical implementation details into our APIs. 

I think it would be best to update `getIncomingEmbeddablePackage` to only support arrays and have a single way to pass one or more embeddables.

cc @ThomThomson

Reply:
💯 I agree that we should do this now rather than introducing the tech debt and fixing it later.